modules/github/__init__.py

```diff
@@ -449,7 +451,9 @@ def commit_comment(self, full_name, data):
         commit = self._short_hash(data["comment"]["commit_id"])
         commenter = utils.irc.bold(data["comment"]["user"]["login"])
         url = self._short_url(data["comment"]["html_url"])
-        return ["[commit/%s] %s commented" % (commit, commenter, action)]
+        comment = utils.irc.bold(data["comment"]["body"])
```

Comment:
we try to keep usage of colour/formatting for GitHub events minimal and **bold** is used everywhere else in the file to denote a username

Reply:
Understood, I'll scrap it.

Reply:
Scrapped!